### PR TITLE
Tanner Dale

### PR DESCRIPTION
**Pull request by TannerDale**

### CODE REVIEW

File: lib/open_mic.rb [hunk 1]
```diff
@@ -1,22 +1,28 @@
+require './lib/user'
+require './lib/joke'
```
Comment (memcmahon) [inline lib/open_mic.rb:2]: Since you are not using these *classes* in this file, you don't need to require them.

File: lib/open_mic.rb [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
-    jokes = @performers.map do |user|
-      user.jokes
-    end.flatten
-    jokes.length != jokes.uniq.length
+    jokes = all_jokes
+
+    jokes != jokes.uniq
+  end
+
+  def all_jokes
```
Comment (memcmahon) [inline lib/open_mic.rb:23]: Nice helper method!  There is a useful method `flat_map` that could also help :)

**Review by memcmahon — COMMENTED** [review]
Great job - this is a pass!

File: lib/open_mic.rb (hunk 2 above)
Comment (TannerDale) [reply lib/open_mic.rb:23]: I know that one but I swear I only remember it like half the time! Thanks

**Pull request closed without merging**